arduino-ide-extension/src/browser/widgets/cloud-sketchbook/cloud-sketchbook-contributions.ts

```diff
@@ -61,6 +64,14 @@ export namespace CloudSketchbookCommands {
     }
   }
 
+  export const SHOW_CLOUD_SKETCHBOOK_WIDGET = Command.toLocalizedCommand(
+    {
+      id: 'arduino-cloud-sketchbook--show-cloud-sketchbook-widget',
+      label: 'Show Cloud Sketchbook Widget',
```

Comment:
```suggestion
      label: 'Show Remote Sketchbook Widget',
```

My understanding is that the official name for this thing is "**Remote Sketchbook**", not "Cloud Sketchbook".

References:

- https://github.com/arduino/arduino-ide/issues/1091
- "**Show/Hide Remote Sketchbook**" command name

Reply:
see [comment](https://github.com/arduino/arduino-ide/pull/1102#issuecomment-1171096122)